ensime_shared/client.py

```diff
@@ -249,7 +249,7 @@ def send_at_position(self, what, useSelection, where="range"):
         self.log.debug('send_at_position: in')
         b, e = self.editor.selection_pos() if useSelection else self.editor.word_under_cursor_pos()
         self.log.debug('useSelection: {}, beg: {}, end: {}'.format(useSelection, b, e))
-        self.send_at_point_req(what, self.editor.path(), b[0], b[1], e[0], e[1], where)
+        self.send_at_range_req(what, self.editor.path(), b[0], b[1], e[0], e[1], where)
```

Comment:
I know, right? the current state is far from ideal, but I tried to reduce the changes not related to the new feature to the minimum. I did take a look at the motivation for such a messy set of functions, and the thing is, that they are like this to hide some inconsistencies from the server's protocol, that arg is used to name the property in the message, some times is called 'range' and some times 'point' but it always carries the 'from' and 'to' that's why I renamed and called `...at_point...` the proper one that actually sends the point.
I'd prefer to leave it like this by now, the sensible refactor to do here would be to introduce a model (with the case classe python equivalent) to represent the server messages instead of cryptic functions which create dics on the fly